_build/builder/BuildDocs.py

```diff
@@ -73,27 +88,19 @@ def Setup(self):
 
     def DoTask(self):
         tmpDir = join(self.buildSetup.tmpDir, "chm")
-        Prepare()
-        #warnings.simplefilter('ignore', DeprecationWarning)
-        sphinx.build_main([
-            None,
-            # '-q',  # be quiet
-            "-a",  # always write all output files
-            "-b", "htmlhelp",
-            "-E",  # Don’t use a saved environment (the structure
-                   # caching all cross-references),
-            "-N",  # Prevent colored output.
-            "-P",  # (Useful for debugging only.) Run the Python debugger,
-                   # pdb, if an unhandled exception occurs while building.
-            "-D", "release=%s" % self.buildSetup.appVersion,
-            "-D", "templates_path=[]",
-            "-d", EncodePath(join(self.buildSetup.tmpDir, ".doctree")),
-            '-v', '-v',  # verbosity, can be given up to three times
-            # write warnings and errors to file
-            # '-w', join('output', 'sphinx_log.txt'),
-            EncodePath(DOCS_SOURCE_DIR),
-            tmpDir,
-        ])
+        WritePluginList(join(self.buildSetup.docsDir, "pluginlist.rst"))
```

Comment:
>I am thinking we should not include a list of plugins in the docs

 I'm not super happy with having one[ plugin list in the docs](http://www.eventghost.net/docs/pluginlist.html) (the bundled plugins) and another[ list in the wiki](http://www.eventghost.net/mediawiki/index.php?title=Plugins) (bundled plugins and 3rd party plugins) but since we already have a plan to make both of those lists obsolete ([plugin repository](https://github.com/EventGhost/EventGhost/issues/4)) it's not something that concerns me much. I'm fine with leaving the system as is for now. I think the energy is better spent on the long term goal of the plugin repository rather than a minor issue that will solve itself in time, possibly before the updated docs even hit the website.

> I do not even know if the list is correct to the plugins that are shipped with EG. I would have to check.

Isn't the plugin list automatically generated? If so it should be complete, with the exception of the plugins that have the `noinclude` flag file.